tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -54,6 +54,10 @@ def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os):
         "centos7": "defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount,"
         "x-systemd.requires=lnet.service",
         "alinux": "defaults,_netdev,flock,user_xattr,noatime",
+        "ubuntu1604": "defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount,"
```

Comment:
optional: to avoid duplications we could have a "default" entry in the dictionary and retrieve the options like this `mount_options.get(os, mount_options["default"])`

Reply:
great idea

Reply:
added